aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -0,0 +1,477 @@
+
+## Certificate and Kerberos authentication
+
+Certificate authentication requires:
+
+* Configuring the server to accept certificates.
+* Adding the authentication middleware in `Startup.Configure`.
+* Adding the certificate authentication service in `Startup.ConfigureServices`.
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddAuthentication(
+        CertificateAuthenticationDefaults.AuthenticationScheme)
+            .AddCertificate();
+    // Other service configuration removed.
+}
+
+public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+{
+    app.UseAuthentication();
+    // Other app configuration removed.
+}
+```
+
+Options for certificate authentication include the ability to:
+
+* Accept self-signed certificates.
+* Check for certificate revocation.
+* Check that the proffered certificate has the right usage flags in it.
+
+A default user principal is constructed from the certificate properties. The user principal contains an event that enables supplementing or replacing the principal. For more information, see <xref:security/authentication/certauth>.
+
+[Windows Authentication](/windows-server/security/windows-authentication/windows-authentication-overview) has been extended onto Linux and macOS. In previous versions, Windows Authentication authentication was limited to [IIS](xref:host-and-deploy/iis/index) and [HttpSys](xref:fundamentals/servers/httpsys). In ASP.NET Core 3.0, [Kestrel](xref:fundamentals/servers/kestrel) has the ability to use Negotiate, [Kerberos](/windows-server/security/kerberos/kerberos-authentication-overview), and [NTLM on Windows](/windows-server/security/kerberos/ntlm-overview), Linux, and macOS for Windows domain joined hosts. Kestrel support of these authentication schemes is provided by the [Microsoft.AspNetCore.Authentication.Negotiate nuget](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Negotiate) package. As with the other authentication services, configure authentication app wide, then configure the service:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddAuthentication(NegotiateDefaults.AuthenticationScheme)
+        .AddNegotiate();
+    // Other service configuration removed.
+}
+
+public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+{
+    app.UseAuthentication();
+    // Other app configuration removed.
+}
+```
+
+Host requirements:
+
+* Windows hosts must have [Service Principal Names](/windows/win32/ad/service-principal-names) (SPNs) added to the user account hosting the app.
+* Linux and macOS machines must be joined to the domain.
+
+  * SPNs must be created for the web process.
+  * [Keytab files](https://blogs.technet.microsoft.com/pie/2018/01/03/all-you-need-to-know-about-keytab-files/) must be generated and configured on the host machine.
+
+See <xref:security/authentication/windowsauth> for more information.
+
+## Template changes
```

Comment:
Should this section also include info about the Razor Class Library change? As by default it's now a blazor class library by default and there is an optional CLI flag `-support-pages-and-views`?

Reply:
Done